src/components/Panel/Panel.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import { Card, Collapse } from 'react-bootstrap'
+import { Icon } from '../Icon'
+
+interface Props {
+  /** Defines the color of the panel */
+  color?: 'primary' | 'secondary' | 'success' | 'warning' | 'danger' | 'info' | 'light' | 'dark'
```

Comment:
this same set of colors are defined in the `ButtonColor` interface in `components/Button/interfaces/ts`. We could probably extract this to a common interface.

Reply:
Where should the common interface reside? a new root folder?

Reply:
Seems fine to me.

Reply:
done